File: problems/problems_3232/Solution.java
package problems.problems_3232;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public boolean canAliceWin(int[] nums) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] nums = jsonArrayToIntArray(inputJsonValues[0]);
        return JSON.toJSON(canAliceWin(nums));
    }
}

File: problems/problems_3232/solution.go
package problem3232

import (
	"encoding/json"
	"log"
	"strings"
)

func canAliceWin(nums []int) bool {

}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}

	return canAliceWin(nums)
}
